nginx/nginx.conf

```diff
@@ -0,0 +1,62 @@
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    sendfile        on;
+
+    keepalive_timeout  65;
+
+    upstream relay {
+        server relay:3000;
+    }
+
+    upstream sentry {
+        server web:9000;
+    }
+
+    server {
+        listen 80;
+        # use the docker DNS server to resolve ips for relay and sentry containers
+        resolver 127.0.0.11 ipv6=off;
+
+        location /api/store/ {
+            proxy_pass http://relay;
+        }
+        location ~ ^/api/\d+/store/$ {
+            proxy_pass http://relay;
+        }
+        location ~ ^/api/\d+/minidump/?$ {
+            proxy_pass http://relay;
+        }
+        location ~ ^/api/\d+/unreal/\w+/$ {
+            proxy_pass http://relay;
+        }
+        location ~ ^/api/\d+/security/$ {
+            proxy_pass http://relay;
+        }
+        location ~ ^/api/\d+/csp-report/$ {
+            proxy_pass http://relay;
+        }
+        location ~ ^/api/\d+/events/[\w-]+/attachments/$ {
+            proxy_pass http://relay;
```

Comment:
Let's replace this with the wildcard version from #8433 as @jan-auer suggested.

Reply:
Yes we should, I double checked in Sentry and I couldn't find any urls ^/api/[1-9]\d*/... the only ones close are /api/0/...